generator/src/googleapis/codegen/languages/php/default/templates/___api_className___/___models_className___.php.tmpl

```diff
@@ -26,6 +26,11 @@ class {{ model.className }} extends {% if model.superClass %}{{ model.superClass
   protected ${{ property.memberName }}DataType = '{{ property.dataType }}';
   {% endif %}
  {% else %}
+ {% if property.annotationType %}
+  /**
+   * @var {{ property.annotationType }}{% if property.dataType == "array" or property.dataType == "map" %}[]{% endif %}
```

Comment:
it's interesting that the `[]` never shows up in our golden tests, which should be covering all the cases. It makes me think this functionality isn't working

Reply:
I think it's working, but you'll only ever see it for arrays or maps of complex types. Like those brackets are rendering this: https://github.com/googleapis/google-api-php-client-services/blob/3f5446122ce80bf4bafaf676969035a6edc452aa/src/DisplayVideo/InventorySource.php#L50-L56

This `{% if %}` block was cribbed from the `@param` stuff below, and you can see that working for method doc blocks (linked above), but complex objects never get declared properties: https://github.com/googleapis/google-api-php-client-services/blob/3f5446122ce80bf4bafaf676969035a6edc452aa/generator/src/googleapis/codegen/languages/php/default/templates/___api_className___/___models_className___.php.tmpl#L22-L30

Presumably because this magic `__get` method has to lazily initialize empty objects: https://github.com/googleapis/google-api-php-client/blob/1530583a711f4414407112c4068464bcbace1c71/src/Model.php#L61-L94

So if this issn't showing up in the golden tests, there's probably nothing in the generator that would trigger that array container type with a primitive PHP type. Not sure I can fix that, though.

Reply:
But `annotationTypes` is only ever set to an instance of `PHP_TYPES` or `PHP_PRIMITIVE_TYPES`, so this could never be a custom class such as `CreativeConfig` above

Reply:
I set `annotationType` on both primitive and complex types: https://github.com/googleapis/google-api-php-client-services/blob/5ec7b7858a8fc89e637ee8892d2ef535973b11b5/generator/src/googleapis/codegen/php_generator.py#L182-L183

Becuase I took `typeHint` to mean the actual type in a method signature vs docblock.

Reply:
Okay, SGTM